src/config.ts

```diff
@@ -0,0 +1,271 @@
+/** Module to define plugin configuration and react on change. */
+import { Logger } from './logger';
+
+const CompositeDisposable = require('atom').CompositeDisposable;
+const logger:Logger = Logger.getInstance();
+
+declare var atom;
+
+/** Plugin configuration visible in Atom editor */
+export const PluginConfig = {
```

Comment:
If you move your configuration to a `configSchema` section in your `package.json` Atom can show your settings without having to transpile+load+run your code.

You can see an example [here](https://github.com/AtomLinter/linter-stylelint/blob/master/package.json#L33-L51), note that this requires Atom v1.4.0 minimum.